.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,23 @@
+name: golangci-lint
+on:
+    push:
+        tags:
+            - v*
+        branches:
+            - master
+            - main
+    pull_request:
+jobs:
+    golangci:
+        name: lint
+        runs-on: ubuntu-latest
```

Comment:
@drpaneas 
```yaml
      - name: Set up LibGL, Mesa & X11 libraries
        run: |
          sudo apt-get --allow-releaseinfo-change update
          sudo apt-get install -y libgtk-3-dev libasound2-dev libxxf86vm-dev
```
else workflow crashes

Reply:
btw, I'm not sure about this line:
`sudo apt-get --allow-releaseinfo-change update`
it was necessary in our CircleCi workflow, because sometimes for some strange reasons it crashed.
however, here it may not be necessary